cf/dimensioncoordinate.py

```diff
@@ -908,6 +1011,164 @@ def roll(self, axis, shift, inplace=False, i=False):
 
         return c
 
+    def set_bounds(self, bounds, copy=True):
+        """Set the bounds.
+
+        .. versionadded:: 3.15.4
+
+        .. seealso: `del_bounds`, `get_bounds`, `has_bounds`, `set_data`
+
+        :Parameters:
+
+            bounds: `Bounds`
+                The bounds to be inserted.
+
+            copy: `bool`, optional
+                If False then do not copy the bounds prior to
```

Comment:
To be clearer by avoiding a double negative:

```suggestion
                If True then copy the bounds prior to
```